.github/workflows/terraform-lint.yaml

```diff
@@ -1,19 +1,16 @@
-name: linting and testing
+name: terraform fmt
 
 on: [pull_request]
 
 jobs:
-  terraform-lint:
+  format:
     runs-on: ubuntu-latest
+    name: Check terraform file are formatted correctly
     steps:
-      - name: Checkout code
+      - name: Checkout
         uses: actions/checkout@v3
 
-      - uses: terraform-linters/setup-tflint@v3
-        name: Setup TFLint
+      - name: terraform fmt
+        uses: dflook/terraform-fmt@v1
```

Comment:
How does this relate to tflint?

Reply:
I moved tflint to pre-commit as the GitHub action frankly didn't work well. I think I left the comment the same though. Terraform-fmt played much nicer in an action so I swapped it in. They do broadly the same stuff so I figured it was a good enough option when coupled with the pre-commits

Reply:
What didn't work well about it?

+1 on the change btw, but worth a bit of detail (perhaps only in this comment change) for what the problem was with tflint

Reply:
It doesn't have a lot of options in the action for restricting where it can run and it wasn't able to apply the changes without a bunch of wonky github interactions. The babenko pre-commits just... like... applied the changes and you were done.